wiiu/vino/vino.ts

```diff
@@ -490,7 +496,7 @@ export default interface VinoAPI {
 	runOliveErrorDialog: (errorCode: number) => boolean;
 
 	/**
-	 * Enable TVii for the TV?
+	 * Plays an video on the TV screen (adding onclick with this function with a <video> tag)
```

Comment:
Please elaborate a bit more? Plays what video? From where? What `onclick`? This hasn't really added much more clarity